app/models/bgeigie_import.rb

```diff
@@ -38,6 +38,15 @@ def self.filter(query)
            OR lower(credits) LIKE :query", query: "%#{query.downcase}%")
   end
 
+  def self.by_user_name(name)
+    users = User.by_name(name)
+    where(user_id: users.map(&:id))
```

Comment:
```suggestion
    where(user_id: users.pluck(:id))
```

could use `pluck` here. But, just saying `where(user: User.by_name(name))` is better because rails would generate quite efficient query.

```
[1] pry(main)> BgeigieImport.where(user: User.by_name('foo'))
=>   BgeigieImport Load (1.3ms)  SELECT "measurement_imports".* FROM "measurement_imports" WHERE "measurement_imports"."type" IN ('BgeigieImport') AND "measurement_imports"."user_id" IN (SELECT "users"."id" FROM "users" WHERE (lower(name) LIKE '%foo%'))
```